packages/Manager/src/main/resources/application-aws.yml

```diff
@@ -2,13 +2,7 @@
 # this enables pulling in configuration from AWS Secrets manager
 spring:
   config:
-    import: aws-secretsmanager:RDSDBConfig # When deployed to AWS this will load from the Secrets
-  datasource:
-    # The property names for the DB values are not ideal. i.e. port should be CONFIG_DATABASE_PORT
-    # But this is reusing the secrets that Gruntworks uses, so going with their format
-    url: jdbc:postgresql://${CONFIG_DATABASE_HOST}:${port}/${dbname}
-    username: ${username}
-    password: ${password}
+    import: aws-secretsmanager:EOPManagerConfig # When deployed to AWS this will load from the Secrets
```

Comment:
This expects the secret to be Key/Value with these
![Screenshot 2023-01-12 at 16 59 41](https://user-images.githubusercontent.com/188683/211972649-d9f87d71-5057-44c2-bf78-d5fea6641bd7.png)

Reply:
Curious why the port and name are in here? Does that suggest we should also include host? Also, are the key names recognised automatically by Spring & Flyway once imported? 

And a little naming pedantry - could this secret contain more than just DB settings in the future? If not, should we name it to scope the intent, e.g. EOPManagerDBConfig?

Reply:
> Curious why the port and name are in here? Does that suggest we should also include host? Also, are the key names recognized automatically by Spring & Flyway once imported?

This was previously overriding this setting in application.yml ...

```
  datasource:
    url: jdbc:postgresql://${CONFIG_DATABASE_HOST:localhost}:${CONFIG_DATABASE_PORT:5432}/${CONFIG_DATABASE_NAME:eop_dev}
    username: ${CONFIG_DATABASE_USERNAME:eop_manager_app_user}
    password: ${CONFIG_DATABASE_PASSWORD:password}
```

And needed to use different names because that the names Gruntworks had used in their secret (called RDSDBConfig) ... now we've created a new secret just for the application, it uses the same naming as application.yml expects. Whereas CONFIG_DATABASE_HOST was getting passed in not via a secret.

.. In hindsight we don't really need the DB port since its just the default, so removed that. But I've left name in there as that is more dynamic